test/org/sagebionetworks/bridge/validators/ScheduleValidatorTest.java

```diff
@@ -102,4 +102,110 @@ public void activityCorrectlyParsesPublishedSurveyPath() {
         assertEquals("2015-01-27T17:46:31.237Z", ref.getCreatedOn());
     }
     
+    @Test
+    public void recurringSchedulesMustHaveCronTriggerOrInterval() {
+        Schedule schedule = new Schedule();
+        schedule.setScheduleType(ScheduleType.RECURRING);
+        try {
+            Validate.entityThrowingException(validator, schedule);
+        } catch(InvalidEntityException e) {
```

Comment:
Add a fail() after the validate to make sure it actually throws. Same for all the rest of the tests.